docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+
+   You must check the following four points using the `kubectl get pod` command before you start the backup operation.
+   * The number of Scalar product pods (You must compare this number with the number after the backup).
+   * The Scalar product pod names in the `NAME` column (You must compare these pod names with the pod names after the backup).
+   * The Scalar product pod status is `Running` in the `STATUS` column.
+   * The restart counts of each pod in the `RESTARTS` column (You must compare these restart counts with the restart counts after the backup).
+
+1. Pause Scalar Product pods using `scalar-admin`.
+
+   Please refer to the [How to use scalar-admin on a Kubernetes environment](./BackupNoSQL.md#how-to-use-scalar-admin-on-a-kubernetes-environment) section in this document for more details on the pause operation.
+
+1. Check the pause completed time.
+
+   You must note the pause completed time to decide the concrete **period** that you can restore data using the PITR feature.
+
+1. Take a backup using the backup feature of each database.
+
+   If you enable the automatic backup and PITR feature, all you have to do is wait for about 10 seconds since the managed databases take backup automatically. This 10 seconds period is the **period** that we can restore data by PITR.
+
+1. Unpause Scalar Product pods using `scalar-admin`.
+
+   Please refer to the [How to use scalar-admin on a Kubernetes environment](./BackupNoSQL.md#how-to-use-scalar-admin-on-a-kubernetes-environment) section in this document for more details on the unpause operation.
+
+1. Check the unpause started time.
+
+   You must note the unpause started time to decide the concrete **period** that you can restore data using the PITR feature.
+
+1. Check the pod status after backup.
+
+   You must check the following four points using the `kubectl get pod` command after you finish the backup operation.
+   * The number of Scalar product pods (You must compare this number with the number before the backup).
+   * The Scalar product pod names in the `NAME` column (You must compare these pod names with the pod names before the backup).
+   * The Scalar product pod status is `Running` in the `STATUS` column.
+   * The restart counts of each pod in the `RESTARTS` column (You must compare these restart counts with the restart counts before the backup).
+
+   **If the two values are different, you must re-try the backup operation all over again.** This is because, if some pods are added or restarted, those pods run with a `unpause` state. The `unpause` state pods cause data inconsistency in the backup data.
+
+1. (DynamoDB only) You need to do additional operations to create a backup if you use the PITR feature of DynamoDB since it restores data with another name table by PITR. You should these additional steps after you complete this creating **period** steps. Please refer to the [Restore database guide](./RestoreDatabase.md#amazon-dynamodb) for more details on these steps.
```

Comment:
```suggestion
8. **(Amazon DynamoDB only)** If you use the PITR feature in DynamoDB, you will need to perform additional steps to create a backup because the feature restores data with another name table by using PITR. For details on the additional steps after creating the exact period in which you can restore the data, please see [Restore a database in a Kubernetes environment](./RestoreDatabase.md#amazon-dynamodb).
```